pkg/probe/probes.go

```diff
@@ -188,8 +195,16 @@ func runPing(_ client.Client) (bool, error) {
 	return true, nil
 }
 
-func ping(target string) (string, error) {
-	cmd := exec.Command("ping", "-c", "1", target)
+func ping(target Route) (string, error) {
+	var dest string
+	// if next hop is IPv6 link-local, we need to append an interface otherwise it is
+	// not clear which interface should be used for communication (e.g. ping test)
+	dest = target.nextHop.String()
+	if target.nextHop.To4() == nil && target.nextHop.IsLinkLocalUnicast() {
```

Comment:
Just a quick followup on Quique's previous comment, it looks like ping with the interface does work fine on ipv4 too. Do we need this logic?

Reply:
LOL I completely missed this, I somehow had imprinted in my memory that it is only for link-local and nothing else. As it indeed works, I am removing this logic

Reply:
Well apparently my memory was correct. After reading deeper (e.g. https://www.rfc-editor.org/rfc/rfc4007#section-11) the syntax with `%` is only correct for link-local IPv6. It may accidentally have worked for us when we tested manually, but in principle it does not work, i.e.

```
{"level":"error","ts":"2023-06-23T16:45:05.060Z","logger":"probe","msg":"error pinging default gateway -> output: ''","error":"failed running ping probe: cmd output: 'ping: 192.168.66.2%bond87: Name or service not known
': exit status 2","errorVerbose":"exit status 2
```

OTOH `ping -I interface address` works for IPv6 link-local too, so may be better if I move to this format